### ISSUE TITLE
Force indent of attributes of components

### ISSUE THREAD

**Issue by vvo** [yannickcr/eslint-plugin-react#181]
Right now I am not sure if any eslint-plugin-react rule can enforce this:

(no error)
```jsx
<MyComponent
  firstProp="value"
  secondProp="value"
/>
```

versus: (error or warning)
```jsx
<MyComponent
firstProp="value"
secondProp="value"
/>
```

It would be good to force to indent attributes so that we know they are under the MyComponent.

If there's no rule and if you can provide some code already dealing with indent, I can have a look

**Comment by yannickcr** [comment]
No rule for now, it was already proposed in #15 but I have not taken the time to do it yet.

One rule dealing with indent is the [eslint/indent](https://github.com/eslint/eslint/blob/master/lib/rules/indent.js) rule but it is a pretty complex one. Luckily I think we do not have to handle so much cases to only check attributes.
